tests/tests_purge_reset.yml

```diff
@@ -1,11 +1,86 @@
 # Test to ensure the logging role successfully runs with
 # logging_purge_confs and logging_reset_confs set to true.
 #
-- name: Ensure that the role runs with default parameters
+- name: Test purge and reset functionality
   hosts: all
-
+  vars:
+    __test_default_files_conf: /etc/rsyslog.d/30-output-files-default_files.conf
   tasks:
-    - name: default run with purge and reset
+    - name: ensure rsyslog is installed
+      package:
+        name: rsyslog
+        state: present
+
+    - name: create some unowned config files
+      copy:
+        dest: "{{ item }}"
+        content: "# this is file {{ item }}"
+        mode: "0444"
+      loop:
+        - "/etc/rsyslog.d/unowned1.conf"
+        - "/etc/rsyslog.d/unowned2.conf"
+
+    - name: modify /etc/rsyslog.conf
```

Comment:
I think this task is not needed since the following task "configure with basics + purge + reset" modifies rsyslog.conf anyway. Actually, the effect of this task is overridden by the next task.

Reply:
But the code won't hit https://github.com/linux-system-roles/logging/pull/265/files#diff-a83292ff37790599fb1a73e2e3a0f69a0f699fdfa00d9af6f57a7b039ea1ddf1R115 and report that rsyslog.conf is modified unless I somehow modify rsyslog.conf first?

Reply:
You are right. This "modify /etc/rsyslog.conf" is needed to execute the block. Sorry for the noise.